homepage-pathway.json

```diff
@@ -25,6 +25,13 @@
         "title": "Upgrading DAML Applications",
         "description": "Learn how to upgrade running DAML aplication and migrate existing data to your new model.",
         "action": "Start"
+        },
+        {
+        "external_link": "https://www.daml.com/learn/fundamental-concepts",
+        "title": "Fundamental DAML concepts",
+        "description": "Understand the 'signatory', 'observer' and 'controller' keywords and learn how you can write secure and privacy respecting DAML applications.",
+        "action": "Start"
         }
```

Comment:
Would move this scenario so that it comes right after the "Testing DAML models" as knowing how to write a scenario is really needed. Noobs really need to go through it.